### ISSUE TITLE
Gigaset Smart Home/Care discontinues its services as of 29 March 2024

### ISSUE THREAD

**Issue by Medic112** [dynasticorpheus/gigasetelements-ha#33]
**Gigaset Smart Home/Care [discontinues](https://www.gigaset.com/hq_en/cms/smart-home-overview.html) its services as of 29 March 2024**
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Hi guys,

maybe you’ve also got a mail from Gigaset, that they will shutdown their Elements cloud on the 29.04.24. Is there any way to host the system locally in the future? 

Greetings

**Comment by setoci** [comment]
Yes, i got it too. That means i can't use the gigaset smart home hardware anymore right? I've invested so much in this system. Now i can put it into trash...

**Comment by Bier01** [comment]
Hi, me too.
A possibility to host it locally would be indeed great, anyhow. 
Martin 

⁣Gesendet mit BlueMail ​

**Comment by Bier01** [comment]
Btw: in my email they talk about shutdown  29.3. !!!!

⁣Gesendet mit BlueMail ​

**Comment by Medic112** [comment]
I think I will just buy some new IKEA Zigbee Sensors and Door/Window Contacts and run them locally in my own built ha setup. Won't buy any cloud based systems again. Sad, Frustrating and time killing... but that's how it is rn.

**Comment by dynasticorpheus** [comment]
> I think I will just buy some new IKEA Zigbee Sensors and Door/Window Contacts and run them locally in my own built ha setup. Won't buy any cloud based systems again. Sad, Frustrating and time killing... but that's how it is rn.

I am taking the same route. Just ordered a SONOFF ZigBee 3.0 USB Dongle Plus, EFR32MG21 which carries the same chipset as the Home Assistant SkyConnect hence assume this will have tier one support in home assistant for long time.

**Comment by setoci** [comment]
Whats about aqara? Seems to be useble offline too.

**Comment by bittner130** [comment]
I'll be going with Aqara, I'm currently using their door and movement sensors (and even light switches) with no issues. I'll be upgrading my thermostats to Aqara thermostats as well. They are missing a siren in their portfolio for replacing my alarm system, so I'll get that from another manufacturer. IKEA remotes are very versatile, I can't recommend them enough.

For a hub I've been using a Deconz ConBee 2 and have had no real issues since using ZHA on Home Assistant instead of other solutions. I can highly recommend looking through [this Site](https://zigbee.blakadder.com) for a Database with known compatible devices of all kinds and different HA-integrations.

Since the Service is closing for good, I want to thank everybody for making this integration! Gigaset was my introduction to smart home and without an integration like this I would have never started my HA journey.

**Comment by dynasticorpheus** [comment]
> Since the Service is closing for good, I want to thank everybody for making this integration! Gigaset was my introduction to smart home and without an integration like this I would have never started my HA journey.

Thanks, kinda sad to see my hobby of creating/maintaining this integration go but since there is nothing we can do about it I am just looking forward to get my **cloudless** setup up and running....

**Comment by jurkov** [comment]
Could someone collect a HAR file of https://app.gigaset-elements.com/ ? HAR Files can be generated with the chrome web browser. Yes, it will contain personal information. If someone could collect the traffic between the browser and the server it would open a door to understand the service. later How to collect a HAR file? https://developer.chrome.com/docs/devtools/network/reference  At your own risk: Visite every site, maybe click every button. And then export the HAR File.

**Comment by dynasticorpheus** [comment]
The end points used are mostly known and part of this integration. What would be needed is the decrypted traffic between sensors & base + base & cloud.

**Comment by jurkov** [comment]
Well, there are instructions https://osmocom.org/projects/misc-dect-hacks/wiki/Gigaset_Elements_Base and firmware files https://www.gigaset.com/hq_en/cms/lp/open-source.html. I couldn't find any API documentation for https://api.gigaset-elements.com/ The file names are promising: 
![image](https://github.com/dynasticorpheus/gigasetelements-ha/assets/923528/529ed309-bb52-4792-80ed-67ba2af415fc)

**Comment by jurkov** [comment]
The missing part would be an MQTT client on the Gigaset Elements Base who can trigger these scripts. Or an REST API triggering the scripts. There are different options.

**Comment by setoci** [comment]
Is there any chance to use the hardware without the cloud service? I already ordered new stuff from aqara 😅

**Comment by dynasticorpheus** [comment]
Bitter sweet D-day where I will be switching to a _Swedish_ cloudless setup. Have to say it looks promising so far! 
```
- HOME ASSISTANT running on LINKSYS WRT32X with OpenWRT Operating System
- SONOFF ZigBee 3.0 USB Dongle Plus, EFR32MG21 Coördinator
- TRÅDFRI Signal repeater
- VALLHORN Wireless motion sensor
- PARASOLL Door/window sensor
- BADRING Water leakage sensor
- NAS-AB02BT Tuya Zigbee Siren
```
![image](https://github.com/dynasticorpheus/gigasetelements-ha/assets/2045116/9374ed63-f1bf-4ba2-a184-11b6776ba636)
